stories/wms.gibs.stories.js

```diff
@@ -0,0 +1,203 @@
+import { renderTilesList } from './storiesUtils';
+
+import {
+  WmsLayer,
+  CRS_EPSG3857,
+  CRS_EPSG4326,
+  BBox,
+  MimeTypes,
+  ApiType,
+  LayersFactory,
+} from '../dist/sentinelHub.esm';
+
+const baseUrl = 'https://gibs.earthdata.nasa.gov/wms/epsg4326/best/wms.cgi';
+// const layerId = 'AIRS_L2_Carbon_Monoxide_500hPa_Volume_Mixing_Ratio_Night';
```

Comment:
Should this be removed?

Reply:
I left it there as a different layer, if someone wants to test... But yeah, it probably doesn't make sense to leave it there, removed, thx.

Reply:
We can create a new story for some different layer someday :)
Resolving.